qualtran/bloqs/chemistry/pbc/first_quantization/__init__.py

```diff
@@ -79,7 +79,4 @@
 ignored.
 """
 
-from .prepare_t import PrepareTFirstQuantization
```

Comment:
What's the strategy here of what is re-exported vs not

Reply:
I think just exporting the big bloqs is sufficient as they're most likely to be interacted with.